gulpfile.js

```diff
@@ -1,26 +1,24 @@
-'use strict';
```

Comment:
If I'm not wrong, Node v0.10 supports `const` when not using `use strict`, so I think you can choose between dropping `use strict` or keeping it and switching all the `const` declarations to `var`. I don't see exactly why both actions are needed. Is there anything I'm missing?